bin/node-template/node/src/service.rs

```diff
@@ -68,7 +68,7 @@ pub fn new_partial(config: &Configuration) -> Result<sc_service::PartialComponen
 		config.transaction_pool.clone(),
 		config.role.is_authority().into(),
 		config.prometheus_registry(),
-		task_manager.spawn_handle(),
+		task_manager.spawn_essential_handle(),
```

Comment:
Makes me wonder if there is any task that is *not* essential.

Reply:
I thought the same :D 

But yes, sometimes, short lived stuff is not essential and is fine to end.